components/esp_hw_support/include/esp_ds.h

```diff
@@ -149,7 +151,9 @@ esp_err_t esp_ds_sign(const void *message,
  * No padding is applied to the message automatically, Please ensure the message is appropriate padded before
  * calling the API.
  *
- * @param message the message to be signed; its length should be (data->rsa_length + 1)*4 bytes
+ * @param message the message to be signed; its length should be (data->rsa_length + 1)*4 bytes, and those
+          bytes must be little endian. Note: this function only applies the raw RSA signing algorithm. It is the
```

Comment:
Can you please add this note in the `note` section.

Reply:
this comment is specific to the message argument. it should go next to message. i've updated it though.